tests/routes/webapp/get_beneficiary_profile_test.py

```diff
@@ -106,6 +106,48 @@ def when_user_has_cancelled_some_offers(self, app):
                 {"domain": "physical", "current": 0.0, "max": 200.0},
             ]
 
+        @pytest.mark.usefixtures("db_session")
+        def when_user_is_created_without_postal_code(self, app):
+            # Given
+            booking = UserFactory(email="wallet_test@email.com", postalCode=None)
+            repository.save(booking)
+
+            # When
+            response = TestClient(app.test_client()).with_auth("wallet_test@email.com").get("/beneficiaries/current")
+
+            # Then
+            assert response.status_code == 200
+
+        @pytest.mark.usefixtures("db_session")
+        def when_user_is_a_pro(self, app):
+            # Given
+            user = UserFactory(
+                email="pro@example.com", postalCode=None, isBeneficiary=False, suspensionReason=None, dateOfBirth=None
+            )
+            user.suspensionReason = None
+            repository.save(user)
+
+            # When
+            response = TestClient(app.test_client()).with_auth("pro@example.com").get("/beneficiaries/current")
+
+            # Then
+            assert response.status_code == 200
+            assert response.json["suspensionReason"] == None
+
+        @pytest.mark.usefixtures("db_session")
+        def when_user_is_a_admin(self, app):
+            # Given
+            user = UserFactory(
+                email="pro@example.com", postalCode=None, isBeneficiary=False, dateOfBirth=None, isAdmin=True
```

Comment:
```suggestion
                email="pro@example.com", postalCode=None, dateOfBirth=None, isAdmin=True
```

`isAdmin=True` implique `isBeneficiary=False` dans la factory, pas besoin de préciser les deux.

Reply:
Ok, je ne savais pas. Il va falloir que je jette un oeil aux factory, je n'ai pas une bonne vision de ce quelles font vraiment.